src/Core/src/Platform/Android/MauiAppCompatActivity.cs

```diff
@@ -29,25 +29,24 @@ protected override void OnCreate(Bundle? savedInstanceState)
 			if (mauiApp.Services == null)
 				throw new InvalidOperationException("App was not initialized");
 
-			var window = mauiApp.GetWindowFor(null!);
+			var mauiContext = new HandlersContext(mauiApp.Services, this);
+			var window = mauiApp.CreateWindowFor(new ActivationState(savedInstanceState, mauiContext));
 
-			window.MauiContext = new HandlersContext(mauiApp.Services, this);
+			window.MauiContext = mauiContext;
 
 			//Hack for now we set this on the App Static but this should be on IFrameworkElement
 			App.Current.SetHandlerContext(window.MauiContext);
 
-			var content = window.Page.View;
+			var content = (window.Page as IView) ?? 
+				window.Page.View;
 
 			CoordinatorLayout parent = new CoordinatorLayout(this);
-			NestedScrollView main = new NestedScrollView(this);
```

Comment:
There's not really a reason to have a scrollview at the root. If user wants a scrollview they can add a scroll view. Thanks to this PR!!